packages/connect/src/utils/debug.ts

```diff
@@ -27,15 +27,33 @@ class Log {
     enabled: boolean;
     css: string;
     messages: LogMessage[];
+    writer: any;
 
-    constructor(prefix: string, enabled: boolean) {
+    constructor(prefix: string, enabled: boolean, logWriter?: any) {
+        console.log('Log constructor !!!');
+        console.log('logWriter', logWriter);
         this.prefix = prefix;
         this.enabled = enabled;
         this.messages = [];
         this.css = typeof window !== 'undefined' && colors[prefix] ? colors[prefix] : '';
+        if (logWriter) {
+            this.writer = logWriter;
+        }
     }
 
     addMessage(level: string, prefix: string, ...args: any[]) {
+        console.log('addMessage');
+        console.log('this.writer', this.writer);
+        // For now we are defining writer only in connect-iframe so
+        // when it is used before in connect it is not declare ????
+        if (this.writer) {
+            this.writer.add({
+                level,
+                prefix,
+                message: args,
+                timestamp: new Date().getTime(),
```

Comment:
yes, we need a timestamp to be able to order logs.

suggestion:
```
this.messages.push(...)
if (this.writer) {
  this.writer.add(this.messages[0])
}
```

nitpick: I was wondering about the difference between new Date().geTime() and Date.now(). Date.now() is [said](https://stackoverflow.com/questions/12517359/performance-date-now-vs-date-gettime) to be a little bit faster. But no need to change it here, new Date.getTime() was already used before this PR

Reply:
I created this PR to improve that https://github.com/trezor/trezor-suite/pull/8764